resources/help.txt

```diff
@@ -411,45 +411,78 @@ Options:
         You cannot use --device and --emulator simultaneously.
     --release - If set, produces a release build. Otherwise, produces a debug build.
 --[/]--
-    
+
 --[debug]--
+Usage:
```

Comment:
Well, `tns debug` will print the debug information specified here. And we do not have a `debug` command. We have `debug|ios` and `debug|android`. Is this a problem?